tests/integration/validate/test_validate_command.py

```diff
@@ -157,8 +157,8 @@ def test_lint_deprecated_runtimes(self, runtime):
             self.assertEqual(command_result.process.returncode, 1)
             self.assertRegex(
                 output,
-                f"\\[\\[E2531: Validate if lambda runtime is deprecated\\] "
-                f"\\(Runtime '{runtime}' was deprecated on.*",
+                f"\\[\\[W2531: Check if EOL Lambda Function Runtimes are used] "
+                f"\\(Runtime \\'{runtime}'\\ was deprecated on.*",
```

Comment:
These `\\` were removed before because they're not needed. You don't need to add them again.

Reply:
Right. Updated. Thanks